contents/handbook/people/team-structure/product-analytics/objectives.mdx

```diff
@@ -1,12 +1,48 @@
-- Objective: Get 80% of active b2b customers to add their marketing website
-  - Why? Increase # events coming in, which increases revenue. Our hypothesis is that marketing websites have 10-100x more events than applications.
-- Objective: Implement most frequently requested features with 3+ happy companies for the feature launch
-  - Key results:
-    - Implement the following:
-      - Multiple breakdowns
-      - Cohort style queries (did x event x times), but within a query itself
-      - Composite events: ‘Any event’, ‘New user’ (users first event), ‘top events’
-      - User composition (ie pie chart where you can select properties)/
-      - Superscale querying - Get 98% of each company’s insights P99 below an acceptable time (to be worked out by the team)
-- Why?
-  - Quality/parity with other product analytics tools. Delighting customers especially power users, will lead to word of mouth growth.
+Input from the execs:
+
+> - Objective: Get 80% of active b2b customers to add their marketing website
+>   - Why? Increase # events coming in, which increases revenue. Our hypothesis is that marketing websites have 10-100x more events than applications.
+> - Objective: Implement most frequently requested features with 3+ happy companies for the feature launch
+>   - Key results:
+>     - Implement the following:
+>       - Multiple breakdowns
+>       - Cohort style queries (did x event x times), but within a query itself
+>       - Composite events: ‘Any event’, ‘New user’ (users first event), ‘top events’
+>       - User composition (ie pie chart where you can select properties)/
+>       - Superscale querying - Get 98% of each company’s insights P99 below an acceptable time (to be worked out by the team)
+> - Why?
+>   - Quality/parity with other product analytics tools. Delighting customers especially power users, will lead to word of mouth growth.
+
+
+
+## Proposed Objective 1: Nail Data Exploration
+
+TODO, but leading up to all the reasons we want universal search, including solving new issues
```

Comment:
what's the overlap between "universal search" and "Cohort style queries (did x event x times), but within a query itself"?

I'm wondering if "universal search" is making me picture the wrong thing